proposals/compatibility-mode.md

```diff
@@ -60,6 +60,8 @@ if desc.dimension is "3d":
   set textureBindingViewDimension to "3d"
 ```
 
+Feature: `texture-views`.
```

Comment:
Kinda makes it sound like you need this to do `texture.createView()`. That's not that far off, so maybe this name is fine.

Reply:
I agree that it's too broad. Riffing:

`unrestricted-texture-views`
`variable-texture-views`
`flexible-texture-views`
`bind-time-texture-view-dimension`
`dynamic-texture-view-dimension`
`unrestricted-texture-view-dimension`

Reply:
I for one like the sound of `flexible-texture-views`.

Reply:
`flexible-texture-views` sounds great

Reply:
Done.